src/adapter/src/coord/timeline.rs

```diff
@@ -347,6 +370,7 @@ impl<S: Append + 'static> Coordinator<S> {
                         Timestamp::minimum(),
                         Timestamp::minimum,
                         *TIMESTAMP_PERSIST_INTERVAL,
+                        *TIMESTAMP_MAX_INCREASE,
```

Comment:
Not this PR but still feels very weird that this logic is duplicated here and in coord.rs, but in coord.rs there's a match on the timeline and here there isn't.

Reply:
Yeah, true. I'll open up an issue and try and fix this for skunkworks.

Reply:
I removed the duplication.